sdk/storage/azblob/blockblob/models.go

```diff
@@ -23,6 +23,14 @@ type Block = generated.Block
 // BlockList - can be uncommitted or committed blocks (committed/uncommitted)
 type BlockList = generated.BlockList
 
+// BlobCopySourceTags - can be 'COPY' or 'REPLACE'
+type BlobCopySourceTags = generated.BlobCopySourceTags
+
+const (
+	BlobCopySourceTagsCOPY    BlobCopySourceTags = "COPY"
```

Comment:
should this be capitalized in the variable name?

Reply:
Yes, this is by-design.  There was a bug in the service requiring the enum values to be all caps.

[OneNote](https://microsoft.sharepoint.com/teams/AzureStorage/_layouts/OneNote.aspx?id=%2Fteams%2FAzureStorage%2FXStore%20Private%20OneNote%2FSTG%20Releases&wd=target%28STG%2073-87%2FSTG%2082.one%7C056FD56A-EDDB-4859-8C40-8D53D761DEC0%2FCopy%20Blob%20from%20URL%5C%2FPut%20Blob%20from%20URL%20copy%20source%7C6CEF33E7-28BA-4F88-8366-4F75A1D1B773%2F%29
onenote:https://microsoft.sharepoint.com/teams/AzureStorage/XStore%20Private%20OneNote/STG%20Releases/STG%2073-87/STG%2082.one#Copy%20Blob%20from%20URL/Put%20Blob%20from%20URL%20copy%20source&section-id={056FD56A-EDDB-4859-8C40-8D53D761DEC0}&page-id={6CEF33E7-28BA-4F88-8366-4F75A1D1B773}&end)

Reply:
yeah but there's no reason we need to be exposing variable names that aren't camel case right?

Reply:
Ya we should probably make the public-facing enum values camel case.

Reply:
Done! I made the variable names camel cased.